scripts/get_webmks_sdk.py

```diff
@@ -0,0 +1,20 @@
+"""Script to get the WebMKS Console option ready for appliance by downloading and extracting SDK."""
+from cfme.utils.appliance import get_or_create_current_appliance
+
+
+def configure_webmks_console(appliance):
+    """Configure VMware Console to use WebMKS."""
+    appliance.server.settings.update_vmware_console({'console_type': 'VMware WebMKS'})
+
+
+def configure_websocket(appliance):
+    """Enable websocket role if it is disabled."""
+    server_settings = appliance.server.settings
+    roles = server_settings.server_roles_db
+    if 'websocket' in roles and not roles['websocket']:
+        server_settings.enable_server_roles('websocket')
+
+if __name__ == '__main__':
+    appliance = get_or_create_current_appliance()
```

Comment:
This should be usable against any appliance passed as a parameter.

Reply:
Do you mean command line argument?

Reply:
Updated